src/webgpu/api/validation/capability_checks/limits/maxTextureDimension2D.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { kLimitBaseParams, makeLimitTestGroup } from './limit_utils.js';
+
+const limit = 'maxTextureDimension2D';
+export const { g, description } = makeLimitTestGroup(limit);
+
+g.test('at_over')
```

Comment:
Same here, and this particular limit actually can get hit from two other places, so we should have TODOs or tests for them: GPUCanvasContext.configure and GPUCanvasContext.getCurrentTexture

Reply:
Added the tests.